### ISSUE TITLE
Issue: Cannot read property 'name' of undefined is causing fields render fail

### ISSUE THREAD

**Issue by wemersonrv** [armincms/nova-tab#1]
Hello. 

When form screen is loaded, it gives me this error in javascript console:
![image](https://user-images.githubusercontent.com/2028673/67586091-31a20e00-f727-11e9-897c-9b7dc00d6936.png)

And the fields of other tabs are rendered in first tab. And if i click on other tabs and return, it fixes... see:

![image](https://user-images.githubusercontent.com/2028673/67586715-a4f84f80-f728-11e9-9aaa-e74cf2790ec2.png)

The current code:
```php
Tab::make('general', function($tab) {
   $tab->group('address', [
      ID::make()->sortable(),
         Text::make('Nome Fantasia', 'nome_fantasia')
            ->rules('nullable'),
   ])->label("Endereços");

   $tab->group('contact', [
      Text::make('Inscrição Estadual', 'ie'),
   ])->label("Contatos");

   $tab->group('other-tab', [
      Text::make('RG', 'rg')
         ->rules('required_if:tipo,Pessoa Física'),
      Text::make('RG Complemento', 'rg_complemento')
         ->rules('required_with:rg'),
   ])->label("Other Tab");

})->fullWidth(),
```

And last. Some fields are not rendering... See the `Other Tab` fields... none of them was rendered:
![image](https://user-images.githubusercontent.com/2028673/67586872-03253280-f729-11e9-84a8-f7fa7d7effb1.png)

Environment: laravel 6.3 and Nova 2.5

**Comment by nibri10** [comment]
I HAVE THE SAME PROBLEM

**Comment by zareismail** [comment]
Thanks for reporting; this bug resolved in version `0.1.2`.
please upgrade into new version and test it. @wemersonrv @nibri10

**Comment by wemersonrv** [comment]
Hello @zareismail. The issue reported is solved, but now, other messages occurs in javascript console. See:

![image](https://user-images.githubusercontent.com/2028673/67622487-ed7b4000-f7f0-11e9-9e40-077781cda9a2.png)

**Comment by zareismail** [comment]
Thanks for using and reporting. 
I installed the package in other projects but I don't see the mentioned error.
This error says that; the `remove` called on the `undefined` thing.
But in underhood; this package is using the `remove` as a `method`, for removing the class from DOM `classList` property; that I think is impossible that DOM properties be the `undefined `. 
Can you give me more information about resolving this error? Please. @wemersonrv
please refer [here](https://www.w3schools.com/jsref/dom_obj_all.asp)

**Comment by wemersonrv** [comment]
> Thanks for using and reporting.
> I installed the package in other projects but I don't see the mentioned error.
> This error says that; the `remove` called on the `undefined` thing.
> But in underhood; this package is using the `remove` as a `method`, for removing the class from DOM `classList` property; that I think is impossible that DOM properties be the `undefined `.
> Can you give me more information about resolving this error? Please. @wemersonrv
> please refer [here](https://www.w3schools.com/jsref/dom_obj_all.asp)

Hello, First of all, forgive-me for my poor english.

Making more checks here, i've seen that this issue occurs when i'm using Tabs with the package [digital-creative/conditional-container](https://novapackages.com/packages/digital-creative/conditional-container) to condition field display to values ​​from other fields, in my case a radio button to allow users to select an option and show/hide fields according to this it.

I think the issue is because this ConditionalContainer, that add or remove items from DOM  because when i use your package without conditional container or vice-versa it's fine. 

I do not understand Vue or DOM manipulation, but i think there must be way to handle this.

Attached, i've sent a resource `Cliente.php` that you can use to test. 
[Cliente.zip](https://github.com/armincms/nova-tab/files/3775757/Cliente.zip)

Any aditional information you need, please call me!

**Comment by nibri10** [comment]
with the same problem ... that Conditional manipulating the DOM and so the tab is giving error ... but the tab needs to be prepared for these changes in the dom

**Comment by zareismail** [comment]
Thanks; for reporting. I'm working on it...

**Comment by zareismail** [comment]
this

> > Thanks for using and reporting.
> > I installed the package in other projects but I don't see the mentioned error.
> > This error says that; the `remove` called on the `undefined` thing.
> > But in underhood; this package is using the `remove` as a `method`, for removing the class from DOM `classList` property; that I think is impossible that DOM properties be the `undefined `.
> > Can you give me more information about resolving this error? Please. @wemersonrv
> > please refer [here](https://www.w3schools.com/jsref/dom_obj_all.asp)
> 
> Hello, First of all, forgive-me for my poor english.
> 
> Making more checks here, i've seen that this issue occurs when i'm using Tabs with the package [digital-creative/conditional-container](https://novapackages.com/packages/digital-creative/conditional-container) to condition field display to values ​​from other fields, in my case a radio button to allow users to select an option and show/hide fields according to this it.
> 
> I think the issue is because this ConditionalContainer, that add or remove items from DOM because when i use your package without conditional container or vice-versa it's fine.
> 
> I do not understand Vue or DOM manipulation, but i think there must be way to handle this.
> 
> Attached, i've sent a resource `Cliente.php` that you can use to test.
> [Cliente.zip](https://github.com/armincms/nova-tab/files/3775757/Cliente.zip)
> 
> Any aditional information you need, please call me!

Hi,

This problem resolved in new release [version 0.3.1](https://github.com/armincms/nova-tab/releases)
thanks for using and reporting.

**Comment by wemersonrv** [comment]
> this
> 
> > > Thanks for using and reporting.
> > > I installed the package in other projects but I don't see the mentioned error.
> > > This error says that; the `remove` called on the `undefined` thing.
> > > But in underhood; this package is using the `remove` as a `method`, for removing the class from DOM `classList` property; that I think is impossible that DOM properties be the `undefined `.
> > > Can you give me more information about resolving this error? Please. @wemersonrv
> > > please refer [here](https://www.w3schools.com/jsref/dom_obj_all.asp)
> > 
> > 
> > Hello, First of all, forgive-me for my poor english.
> > Making more checks here, i've seen that this issue occurs when i'm using Tabs with the package [digital-creative/conditional-container](https://novapackages.com/packages/digital-creative/conditional-container) to condition field display to values ​​from other fields, in my case a radio button to allow users to select an option and show/hide fields according to this it.
> > I think the issue is because this ConditionalContainer, that add or remove items from DOM because when i use your package without conditional container or vice-versa it's fine.
> > I do not understand Vue or DOM manipulation, but i think there must be way to handle this.
> > Attached, i've sent a resource `Cliente.php` that you can use to test.
> > [Cliente.zip](https://github.com/armincms/nova-tab/files/3775757/Cliente.zip)
> > Any aditional information you need, please call me!
> 
> Hi,
> 
> This problem resolved in new release [version 0.3.1](https://github.com/armincms/nova-tab/releases)
> thanks for using and reporting.

Hi, tested and is fine now, except for a litthe detail:

In details page... The fields i used with Conditional Container in another tabs are duplicated in 1st tab. See below:

![image](https://user-images.githubusercontent.com/2028673/67984390-5efa2a80-fc05-11e9-9734-a07d77528fe0.png)

These Fields were used only on tab `Documentos` but are been duplicated to first tab
